hypervisor/vm.go

```diff
@@ -358,7 +358,7 @@ func (vm *Vm) AddRoute() error {
 
 func (vm *Vm) AddNic(info *api.InterfaceDescription) error {
 	client := make(chan api.Result, 1)
-	vm.sendGenericOperation("CreateInterface", func(ctx *VmContext, result chan<- error) {
+	vm.handleGenericOperation("CreateInterface", func(ctx *VmContext, result chan<- error) {
 		go ctx.AddInterface(info, client)
```

Comment:
could you also remove vm.handleGenericOperation()?
we can use ctx.AddInterface() directly.